packages/core/src/indexing/postgres/migrations/mid-schema-verification.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Valid Postgres table structure for mid tables
+ * Used to verify table integrity during node startup
+ */
+
+export const validTableStructure = {
+  stream_id: {
+    type: 'character varying',
+    maxLength: 255,
+    nullable: false,
+    defaultValue: null,
+  },
+  controller_did: {
+    type: 'character varying',
+    maxLength: 255,
+    nullable: false,
+    defaultValue: null,
+  },
+  last_anchored_at: {
+    type: 'timestamp with time zone',
+    maxLength: null,
```

Comment:
Does `maxLength` matter here for the timestamp type?

Reply:
it doesn't and wasn't explicitly set by me but is rather a default value set by the ORM during creation.